graf3d/x3d/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(libname X3d)
 
 include_directories(${X11_INCLUDE_DIR})
 
-ROOT_GENERATE_DICTIONARY(G__X3D TViewerX3D.h TX3DFrame.h MODULE ${libname} LINKDEF LinkDef.h DEPENDENCIES Graf3d Gui)
```

Comment:
I agree, but that's out of the scope of this particular PR. It can be addressed in a new PR later.